noxfile.py

```diff
@@ -0,0 +1,8 @@
+import nox
```

Comment:
For local support; we can make `nox` as part of `dev-requirements.txt`

Reply:
@amolkahat We also need updates to README file